municipal_finance/static/javascript/table.js

```diff
@@ -800,7 +832,22 @@
                 var data = muni_data ? muni_data[functions[f].code] : null;
                 this.renderMuniValues(muni, data, tr);
               }
-            } else {
+            } else if (CUBE_NAME == 'capital_v2') {
+              var data = [];
+              if (muni_data){
+                Object.keys(aggregate_columns).forEach(column => {
+                  var row = "";
+                    for (var f = 0; f < muni_data.length; f++) {
```

Comment:
Indenting is off. We should look to use a formatter rather than relying on hand formatting.

Reply:
I adjusted the formatting here. I try to fix formatting in table.js where I touch code but it's probably a good idea to do one big format and then it's good